templates/web/nottinghamshirepolice/footer_extra.html

```diff
@@ -0,0 +1,128 @@
+<footer class="main-footer">
+  <div class="container">
+    <div class="main-content">
+      <div class="first-column">
+        <img src="/cobrands/nottinghamshirepolice/images/logo.svg" width="265" height="80" alt="Nottinghamshire PCC logo" load="lazy">
+        <img src="/cobrands/nottinghamshirepolice/images/inmediate-justice-logo.png" width="265" height="100" alt="Inmediate Justice logo" load="lazy">
```

Comment:
Typos here too (you'll need to change the name of the .png file) - should be 'immediate' :-)